ubuntu/ubuntu-22.x/ubuntu-22.04-hpc/install.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -ex
 
+export GPU=$1
```

Comment:
Should we validate the incoming param? As it is, I could send in an invalid param (even "nvidia" in lowercase) and not know for 45 min as the installation script would proceed as normal and skip over all of the GPU stuff.

Reply:
I've added a check for this. By default GPU will be set to NVIDIA unless "AMD" is passed in, in which case it will be set to AMD.